src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAbcdProductsPriority.java

```diff
@@ -52,8 +53,8 @@ public void adsUAPFamilyAdIsNotDisplayedOnFeaturedVideoPage() {
   }
 
   private void verifyNoUAPSizesInSlot(AdsBaseObject ads, String slot) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "verifyNoUAPSizesInSlot" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)